src/rebuild.ts

```diff
@@ -42,6 +44,8 @@ class Rebuilder {
     this.ABI = nodeAbi.getAbi(electronVersion, 'electron');
     this.prodDeps = extraModules.reduce((acc, x) => acc.add(x), new Set());
     this.rebuilds = [];
+    this.realModulePaths = [];
+    this.realNodeModulesPaths = [];
```

Comment:
See my comment below about using sets, all we need is a concept of "has been used before", we don't need to store a value with the path so maps will be less efficient than using a raw set